Nodejs/Product/Nodejs/Debugger/NodeDebugger.cs

```diff
@@ -1239,6 +1241,37 @@ public NodeDebugger(Uri debuggerEndpointUri, int id)
             return true;
         }
 
+        /// <summary>
+        /// Get the original file name to map to.
+        /// </summary>
+        /// <param name="javaScriptFileName">JavaScript compiled file.</param>
+        /// <param name="stackFrame">The current stack frame.</param>
+        internal string GetOriginalFileName(string javaScriptFileName, NodeStackFrame stackFrame, SourceMapInfo mapping) {
+
+            SourceMapInfo tempMapping;
+            string originalFileName = String.Empty;
+            bool mappingReady = false;            
+
+            if (stackFrame != null) {
+                tempMapping = SourceMapper.MapToOriginal(javaScriptFileName, stackFrame.Line, stackFrame.Column);
+
+                if (tempMapping != null) {
+                    originalFileName = tempMapping.FileName;
```

Comment:
tempMapping.FileName should have a value if tempMapping did. Cleaned an unnecessary condition check from the previous code.